libs/core/carousel/carousel.component.ts

```diff
@@ -304,6 +316,24 @@ export class CarouselComponent implements AfterContentInit, AfterViewInit, After
     ngAfterContentInit(): void {
         // On carousel load, display first slide + number of slide visible
         this.currentActiveSlidesStartIndex = 0;
+        this.currentActiveSlidesEndIndex = this._visibleSlidesNumericCount - 1;
+        this.currentActiveSlidesIds = [];
+        this.totalSlides = this.slides.length;
+
+        this.slides.forEach((slide, index) => {
+            const isActive = index >= this.currentActiveSlidesStartIndex && index <= this.currentActiveSlidesEndIndex;
+
+            slide.ariaSetsize.set(this.totalSlides);
+            slide.ariaPosinset.set(index + 1);
+            slide.ariaHidden.set(!isActive);
+            slide.ariaSelected.set(isActive);
+
+            if (isActive) {
+                this.currentActiveSlidesIds.push(slide.id);
+            }
+        });
+
+        this.ariaActivedescendant = this.currentActiveSlidesIds.join(' ');
```

Comment:
this is necessary as the Carousel can have more than 1 visible slides at the same time and the string represents the ids of the visible items separated by a space: "item-1 item-2 item-3".